kpatch-build/insn/inat-tables.c

```diff
@@ -2,7 +2,7 @@
 /* Do not change this code. */
 
 /* Table: one byte opcode */
-const insn_attr_t inat_primary_table[INAT_OPCODE_TABLE_SIZE] = {
+static const insn_attr_t inat_primary_table[INAT_OPCODE_TABLE_SIZE] = {
```

Comment:
There are a few ways to invoke the static parsers:  For the earlier (merged) patchset round I set the ```CHECK``` variable that the kernel's Makefile reads.  For this patchset, I set ```CC=~/smatch/cgcc``` to run the analyser over _all_ of the code.

As far as I can see, there is no easy way to pass to smatch/sparse a list of source files to ignore nor any inline annotation to instruct the parsers to ignore a particular source code line.

I suppose our Makefile(s) could be modified in a way to accomplish this, but that seems like overkill.  I think the best compromise would be to drop ```inat-tables.c``` from this PR... smatch comes with a ```new_bugs.sh``` script that can report just the _new_ warnings if that's what we're ultimately interested in.

We could also push the same fixes for ```inat-tables.c``` upstream and then resync.  (I think this file is auto-generated, but perhaps its owner would take these fixups.)

Let me know and I'll push out a v2.